context.go

```diff
@@ -921,6 +949,8 @@ func (c *Context) Pow(d, x, y *Decimal) (Condition, error) {
 
 	// decNumber sets the precision to be max(x digits + exponent, c.Precision)
 	// + 4. 6 is used as the exponent digits.
+	// TODO(mjibson): what is decNumber, and what is the source of the +4?
```

Comment:
OK on decNumber.
The 4 and 6 I'm less OK with, perhaps we should leave a todo here making it clear that these are cargo-cult magic constants (since the other magic constants are clearly documented).